spec/install/gemfile/git_spec.rb

```diff
@@ -82,7 +82,23 @@
         gem "foo", "1.1", :git => "#{lib_path("foo-1.0")}"
       G
 
-      expect(out).to include("Source contains 'foo' at: 1.0")
+      expect(out).to include("Source contains 'foo' at: 1.0 ruby")
+    end
+
+    it "complains with platform (if not ruby) if pinned specs don't exist in the git repo" do
```

Comment:
Hm okay, so what is the functional process for having multiple versions in the source when the git value refers to a specific directory with just a gemspec and basic lib file? I.e. `lib_path("only_java-1.0-java")` is where it looks for sources, so what does having multiple versions in that directory entail?